x-pack/plugin/src/test/resources/rest-api-spec/api/data_frame.get_data_frame_transform.json

```diff
@@ -0,0 +1,18 @@
+{
+  "data_frame.get_data_frame_transform": {
+    "documentation": "TODO",
+    "methods": [ "GET" ],
+    "url": {
+      "path": "/_data_frame/transforms/{transform_id}",
+      "paths": [ "/_data_frame/transforms/{transform_id}"],
+      "parts": {
+        "transform_id": {
+          "type": "string",
+          "required": false,
+          "description": "The id of the transforms to get, '*' implies get all transforms"
```

Comment:
How about `_all`?  If that's a synonym for `*` maybe we should mention that too.

If that's not implemented then that's an inconsistency with other APIs that we should fix at some point.  Don't fix it as part of this PR but raise an issue so someone can add `_all` in the future.